lib/container_server.go

```diff
@@ -701,7 +700,7 @@ func (c *ContainerServer) AddSandbox(sb *sandbox.Sandbox) {
 	c.state.sandboxes.Add(sb.ID(), sb)
 
 	c.stateLock.Lock()
-	c.state.processLevels[selinux.NewContext(sb.ProcessLabel())["level"]]++
+	c.addSandboxPlatform(sb)
```

Comment:
Not crazy about the method name.  The method is label or SELinux specific.  Why not c.allocateSecurityLabel(sb), c.freeSecurtyLabel(sb)?

Reply:
I *figured* that there may be platform dependent setup needed. This allow an abstraction point. Currently on linux that is only for selinux

Reply:
Ok.  More of a Nit than anything else.